controllers/quay/quayregistry_controller.go

```diff
@@ -302,9 +301,11 @@ func (r *QuayRegistryReconciler) checkPostgresUpgradeStatus(
 			}
 
 			// If user has set POSTGRES_UPGRADE_RETAIN_BACKUP=true, then we should not delete the old PVC
-			if os.Getenv("POSTGRES_UPGRADE_RETAIN_BACKUP") == "true" {
-				continue
-			}
+			//if os.Getenv("POSTGRES_UPGRADE_RETAIN_BACKUP") == "true" {
```

Comment:
Instead of removing this completely, can we just reverse the flag? To something like POSTGRES_UPGRADE_DELETE_BACKUP ?

Reply:
sure, that's a good idea